Source/VolunteerReporting/Web.Angular/src/app/case-report/case-report-list/case-report-list.component.ts

```diff
@@ -33,10 +34,19 @@ export class CaseReportListComponent implements OnInit {
 
     basicFilter: string = "all";
     
+    fields: Array<string> = [
+        "id", "status", "dataCollectorId", "dataCollectorDisplayName",
+        "healthRiskId", "healthRisk", "message",
+        "numberOfFemalesOver5", "numberOfFemalesUnder5",
```

Comment:
I have just renamed the age properties. Ensure that these are correct after merge with my PR :)

Reply:
Will do